lib/database_connection.rb

```diff
@@ -0,0 +1,16 @@
+require 'pg'
+
+class Database_Connection
```

Comment:
Very wise to use a separate file for the connecting features, makes the rest of your code (peeps & users) much more simplified and clear.